### ISSUE TITLE
new MobileBridge presence on Bridge Foundry Discourse forum

### ISSUE THREAD

**Issue by kmolo** [mobilebridge/organizing#8]
Hi everyone,

I've opened two categories on Bridge Foundry's Discourse forum for us:

http://discourse.bridgefoundry.org/t/about-the-mobilebridge-ios-help-category/

http://discourse.bridgefoundry.org/t/about-the-mobilebridge-android-help-category

...and I'd like to invite you all to join.

iOS /cc @glenna @LauraKirby @timothy1ee @ultrasaurus @rajatb @mtitolo @yskamoona @devhut @steffi06 @stansey @mmesarina @tjgrathwell @imju @StephanieSzeto @joker2600 @durhamka @mludowise @joekim @aparnarsjain @shawnyapa @xenadu @gwengrid @dvidsilva @dmtsui @kristen @mrh-is @salilmalkan

Android /cc @nesquena @rogerhu @anubharadwaj @tensory @BrainScraps @jangelroa @kgleong @mdcramer @martasmith @nidhi1608 @paulinar @sonamsharma @vibhorB @mydiemho

**Comment by jeffreybergier** [comment]
@kmolo is there a way to subscribe to that forum to be notified if someone posts? Is there a digest subscription option?

**Comment by glenna** [comment]
@jeffreybergier yeah, you can select your email preferences from the account menu.

**Comment by kmolo** [comment]
There isn't a how-to guide for Discourse, just the [meta.discourse.org](https://meta.discourse.org/) forums.

Also, posted a link to talk that Jeff Atwood gave about Discourse/StackOverflow to the SF Community Managers meetup:

http://discourse.bridgefoundry.org/t/link-to-jeff-atwood-talk-about-discourse-as-a-platform/15

...a great talk for understanding the philosophy & some of the cool features behind Discourse.  Great talk about community management in general -- I've sent it along to others who aren't using Discourse for their communities.

**Comment by kmolo** [comment]
Also starting a conversation about *Bridge-specific badges to create on the Discourse forum.

Here are the pre-existing ones:  http://discourse.bridgefoundry.org/badges

Setting up new badges is fairly easy, and here are some possibilities:

 - Workshop Volunteer
 - Workshop Organizer

...and we could get more granular:

 - Android Dev Env Installed
 - iOS Dev Env Installed
 - First [Android | iOS | Clojure | Rail] App Built
 - First Android App in Google Play Store
 - First iOS App in Apple App Store

Your thoughts on these? Others you would you suggest?
